### PR TITLE
Using Routes to Navigate between pages

### PR DESCRIPTION
**Pull request by prius2055**
### Using Routes to Navigate between pages

**I have implemented the following functionalities:**

- Created a Home component in the pages directory
- Created a Qoute component in the pages directory
- Created a Navbar component
- Linked the Navbar links with the corresponding documents in the pages directory

### CODE REVIEW

File: src/components/NavBar.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  margin-left: 30px;
+  list-style: none;
+  padding: 5px;
+  font-weight: 500;
+}
+
+.navbar a {
+  text-decoration: none;
+
+  /* color: black; */
```
Comment (rustam-tolipov) [inline src/components/NavBar.css]: - [ ] To make your code clean, kindly remove, unused commented codes from your file!

File: src/pages/Calculator.js [hunk 2]
```diff
@@ -42,8 +42,10 @@ const Calculator = () => {
 
   return (
     <div className='calculator'>
-      <CalculatorDisplay display={result} />
+      <h3>Let&apos;s do some math!</h3>
```
Comment (rustam-tolipov) [inline src/pages/Calculator.js:44]: - [ ] Please remove this unused [code](https://github.com/prius2055/Math-magicians/pull/5/files#diff-518ab039f8a31bc368b8b8e69ec8cdb3e1fa5849fcb27ea14c57ec2b3cefc946L2:~:text=//%20import%20propTypes%20from%20%27prop%2Dtypes%27%3B), and this unused [code](https://github.com/prius2055/Math-magicians/pull/5/files#diff-518ab039f8a31bc368b8b8e69ec8cdb3e1fa5849fcb27ea14c57ec2b3cefc946L2:~:text=//%20Calculator.propTypes%20%3D%20%7B,//%20%7D%3B) as well, thank you!

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @prius2055,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Well-documented readme_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Calculator.js (hunk 2 above)
Comment (prius2055) [reply src/pages/Calculator.js:44]: I have cleaned the unused codes from Calculator and Navbar components

File: src/components/NavBar.css (hunk 1 above)
Comment (rloterh) [reply src/components/NavBar.css]: well resolved! ✔️

File: src/pages/Calculator.js (hunk 2 above)
Comment (rustam-tolipov) [reply src/pages/Calculator.js:44]: In Calculator.js, line 70

```javascript
// Calculator.propTypes = {
//   onButtonClick: propTypes.func.isRequired,
// };
``` 

this one also removed, right?

Comment (rloterh) [reply src/pages/Calculator.js:44]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Prince,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉 💯 🌟 

## Highlights 👍
- Good job implementing recommended changes!

Cheers..Happy coding! 💻  🍷  🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Calculator.js (hunk 2 above)
Comment (prius2055) [reply src/pages/Calculator.js:44]: @Rustamxon7 Thanks for that. Eagle eye!

Comment (rustam-tolipov) [reply src/pages/Calculator.js:44]: Hope it works for your benefit :)

Comment (prius2055) [reply src/pages/Calculator.js:44]: Spacibo Bratan!

Comment (rustam-tolipov) [reply src/pages/Calculator.js:44]: Pojalusta!

**Pull request merged**